tests/setup.js

```diff
@@ -32,6 +32,28 @@ jest.mock('@sentry/electron/renderer', () => ({
   init: jest.fn(),
 }));
 
+class FakeBroadcastChannel {
+  listeners = [];
+
+  addEventListener = (_, callback) => {
+    this.listeners.push(callback);
+  };
+
+  postMessage = (message) => {
+    for (const listener of this.listeners) {
+      listener(new MessageEvent('message', { data: message }));
+    }
+  };
+}
+
+global.BroadcastChannel = class Singleton {
+  static instance = new FakeBroadcastChannel();
+
+  constructor(_) {
+    return Singleton.instance;
+  }
+};
```

Comment:
```suggestion
class FakeBroadcastChannel extends EventTarget {
  constructor(channelName) {
    super();
    this.name = channelName;
  }

  postMessage(message) {
    this.dispatchEvent(new MessageEvent('message', { data: message }));
  }
}

global.BroadcastChannel = class Singleton {
  static channels = new Map();

  constructor(channelName) {
    if (!Singleton.channels.has(channelName)) {
      Singleton.channels.set(
        channelName,
        new FakeBroadcastChannel(channelName),
      );
    }
    return Singleton.channels.get(channelName);
  }
};
```

Can extend from `EventTarget` here to handle most of the actual implementation, and flesh out `global.BroadcastChannel` to act more like the spec `BroadcastChannel`.